Mario PM Dair + Galaxy spin

Nakedkoops opened the pull request:
This pr has the complete versions both the galaxy spin and the Pm dair.

Again, extra thanks to MaroonRed for the star spin sound effect used in both ult-s and this  move, Askew for helping me remove the Fludd model, and Zonic for both helping me with porting the sound effect and play-test this move to find both bugs and broken tech. 
None of this would be possible without them.
 
### Galaxy spin:
- Startup 6 frames
- Lingering hitbox frames 6 - 22 (16 frames total)
- Cancel frame 42 (Grounded)
- Cancel frame 47 (Aerial)
- When used in the air, allow you to gain slight extra height (only used once per airtime)
- Damage: 8.0
- Angle: 70
- KGB: 45
- BKB: 88
- Hitstun: 2.3

Edit: If you do wall jump up b, you cannot do another up b, in the same air time.

### PM Dair: 
- Startup 8 frames 
- Lingering hitbox frames 8-29 (21 frames total)
- Aerial Cancel frame 42-43
- Land Cancel frame 10
- Damage: 1.1 per hit
- Angle: 367
- Total multi-hits: 8 
- Total Damage (if all hits connect): 8.8

PR assets: 
[hdr-assest-dev.zip](https://github.com/HDR-Development/HewDraw-Remix/files/12270265/hdr-assest-dev.zip)

Also to the devs: if you are wondering why I'm merging from spin -> dair and not dair -> spin, its because there were complications when merging with latter and idk what causes its. For the sake of simplicity, the dair pr is going to feature both of his change. You can close the galaxy spin pr if yall want.

Video showcase:
https://github.com/HDR-Development/HewDraw-Remix/assets/115908645/becbc6cd-422a-4596-9739-59a630f76d03

https://github.com/HDR-Development/HewDraw-Remix/assets/115908645/8662c012-b4fc-4847-932c-50bbdcd0d79e

https://github.com/HDR-Development/HewDraw-Remix/assets/115908645/f3a5fabb-2d3c-45f7-841a-f1f6e678b900

https://github.com/HDR-Development/HewDraw-Remix/assets/115908645/b4448708-e54e-4293-904f-e3654c8decc5

https://github.com/HDR-Development/HewDraw-Remix/assets/115908645/8662c012-b4fc-4847-932c-50bbdcd0d79e

Nakedkoops committed:
Update specials.rs

Nakedkoops committed:
Merge remote-tracking branch 'upstream/dev' into Mario-Firebrand-Input-change

Nakedkoops committed:
Super Mario Galaxy Spin (Prototype

SMG spin.

Nakedkoops committed:
Update specials.rs

Nakedkoops committed:
Merge branch 'dev' into Super-Mario-Galaxy-spin

Nakedkoops committed:
Fucktons of changes

graphics
sound effects
slight adjustments to the move

Nakedkoops committed:
Final touchups

Comments,
Minor adjustments,
etc

Nakedkoops committed:
touchups to complete review

Nakedkoops committed:
Merge remote-tracking branch 'upstream/dev' into Super-Mario-Galaxy-spin

Nakedkoops committed:
Jump cancel and slight hitbox adjustments

idk guess

Nakedkoops committed:
linear interpolation on grounded spin

Nakedkoops committed:
Merge branch 'dev' into Super-Mario-Galaxy-spin

Nakedkoops committed:
Update specials.rs

Nakedkoops committed:
Update other.rs

Nakedkoops committed:
Update specials.rs

Nakedkoops committed:
Removal of jump cancel & other adjusments

Nakedkoops committed:
Merge remote-tracking branch 'upstream/dev' into Super-Mario-Galaxy-spin

Nakedkoops committed:
Merge remote-tracking branch 'upstream/dev' into Super-Mario-Galaxy-spin

Nakedkoops committed:
Update motion_patch.yaml

Nakedkoops committed:
PM Dair go broooommmm

Nakedkoops committed:
effects n' shit

Nakedkoops committed:
pee pee poo poo

Nakedkoops committed:
something

Nakedkoops committed:
sfx changes

hdr-balance-backroom reviewed (changes requested):
- Please move these changes into the Galaxy spin PR as they are meant to be in tandem

Nakedkoops committed:
Merge branch 'Mario-PM-Dair' into Super-Mario-Galaxy-spin

Nakedkoops committed:
Merge remote-tracking branch 'upstream/dev' into Super-Mario-Galaxy-spin

Nakedkoops committed:
Fast fall removeal on spin and small touchups

Nakedkoops committed:
updates

if you do galaxy spin you cannot do wall jump up b and if you do wall jump up b you can’t do galaxy spin

Nakedkoops committed:
Merge remote-tracking branch 'upstream/dev' into Mario-PM-Dair

Nakedkoops committed:
Merge branch 'Mario-PM-Dair' of https://github.com/Nakedkoops/HewDraw-Remix into Mario-PM-Dair

Nakedkoops committed:
Update opff.rs

Recovery changes

hdr-balance-backroom reviewed (dismissed):
- Mario is not put into hitlag properly when connecting with Galaxy spin. After fixing this, the hitlag mul should probably be decreased to 1.5
- Dair has too much hitstun, it can link into itself even at 0%. Change the knockback to where the frame advantage is around +3 at 0% and +7 at 100%. In addition, please decrease the angle to 365 and the sdi mul to 1.0

SkewedAskew committed:
Merge branch 'dev' into Mario-PM-Dair

SkewedAskew committed:
Feedback adjustments
